<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Expression</class>
 <widget class="QWidget" name="Expression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_anger">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/anger.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_disdain">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/disdain.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_excited">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/excited.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/fear.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_normal">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/normal.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/sad.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_anger_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/anger.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_anger_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/anger.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_anger_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/expression/build/Desktop_Qt_6_5_3_MinGW_64_bit-Debug/anger.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_start">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>愤怒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>不屑</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>兴奋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>惊恐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>难过</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>右转</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>静态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>左转</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
